net.ankiweb.Anki.json

```diff
@@ -83,104 +125,66 @@
           "url": "https://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.9.2/PyQt5_gpl-5.9.2.tar.gz",
           "sha256": "c190dac598c97b0113ca5e7a37c71c623f02d1d713088addfacac4acfa4b8394"
         }
-      ],
-      "modules": [
+      ]
+    },
+    {
+      "name": "portaudio",
+      "sources": [
         {
-          "name": "sip",
-          "sources": [
-            {
-              "type": "archive",
-              "url": "https://sourceforge.net/projects/pyqt/files/sip/sip-4.19.7/sip-4.19.7.tar.gz",
-              "sha256": "25b50d29dd4b72965e7980c41e3320e460eff481a177beeddebf8c3be84b8cde"
-            }
-          ],
-          "buildsystem": "simple",
-          "build-commands": [
-            "python3 configure.py --bindir=/app/bin --destdir=/app/lib/python3.5/site-packages --incdir=/app/include --pyidir=/app/lib/python3.5/site-packages --sipdir=/app/share/sip",
-            "make",
-            "make install"
-          ],
-          "cleanup": [
-            "/bin",
-            "/include"
-          ]
+          "type": "archive",
+          "url": "http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz",
+          "sha256": "f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513"
         }
+      ],
+      "cleanup": [
```

Comment:
This portaudio cleanup is different from the original, so the output from the build differs in that respect. any chance you could limit this commit to purely stylistic changes, and move the cleanups to separate commits?

Reply:
I don’t think this should change the build, just a simplification of the cleanup. Well, it won’t change the build, if the previous cleanup actually worked. `/app` is added automatically, so I’m not sure if this would double it or not, I didn’t think of that. I’ll check.

Reply:
Yeah, the previous cleanup was broken, it didn’t clean anything, so this _does_ actually change the build.

Still, the change is purely cosmetic (along with a megabyte shaved off the Flatpak), it doesn’t affect the app function in any way, I’d just leave it TBH. But if you insist, I’ll try to summon my git-rebase-fu.

Reply:
Summoned.